vector.hpp

```diff
@@ -159,17 +159,17 @@ class Vector3D {
 	/**
 	 * @brief Computes the pitch angle of the vector.
 	 */
-	NumericType pitch() const { return atan2(Z_coord, magnitudeXY()) * rad2DegFactor; }
```

Comment:
This change isn't safe without tests that demonstrate that it doesn't break anything.  If it's causing an error, please provide a test case that demonstrates the error and shows that this fixes it.